openlibrary/templates/languages/language_list.html

```diff
@@ -1,17 +1,17 @@
 $def with (classes='')
 
 <ul class="locale-options $classes">
-  <li><a href="#" lang="cs" data-lang-id="cs" title="$_('Czech')">Čeština (cs)</a></li>
-  <li><a href="#" lang="de" data-lang-id="de" title="$_('German')">Deutsch (de)</a></li>
-  <li><a href="#" lang="en" data-lang-id="en" title="$_('English')">English (en)</a></li>
-  <li><a href="#" lang="es" data-lang-id="es" title="$_('Spanish')">Español (es)</a></li>
-  <li><a href="#" lang="fr" data-lang-id="fr" title="$_('French')">Français (fr)</a></li>
-  <li><a href="#" lang="hr" data-lang-id="hr" title="$_('Croatian')">Hrvatski (hr)</a></li>
-  <li><a href="#" lang="it" data-lang-id="it" title="$_('Italian')">Italiano (it)</a></li>
-  <li><a href="#" lang="pt" data-lang-id="pt" title="$_('Portuguese')">Português (pt)</a></li>
-  <li><a href="#" lang="hi" data-lang-id="hi" title="$_('Hindi')">हिंदी (hi)</a></li>
-  <li><a href="#" lang="sc" data-lang-id="sc" title="$_('Sardinian')">Sardu (sc)</a></li>
-  <li><a href="#" lang="te" data-lang-id="te" title="$_('Telugu')">తెలుగు (te)</a></li>
-  <li><a href="#" lang="uk" data-lang-id="uk" title="$_('Ukrainian')">Українська (uk)</a></li>
-  <li><a href="#" lang="zh" data-lang-id="zh" title="$_('Chinese')">中文 (zh)</a></li>
+  <li><a href="#" lang="cs" data-lang-id="cs" title="$_('Czech')">$("Čeština (cs)")</a></li>
+  <li><a href="#" lang="de" data-lang-id="de" title="$_('German')">$("Deutsch (de)")</a></li>
+  <li><a href="#" lang="en" data-lang-id="en" title="$_('English')">$("English (en)")</a></li>
+  <li><a href="#" lang="es" data-lang-id="es" title="$_('Spanish')">$("Español (es)")</a></li>
+  <li><a href="#" lang="fr" data-lang-id="fr" title="$_('French')">$("Français (fr)")</a></li>
+  <li><a href="#" lang="hr" data-lang-id="hr" title="$_('Croatian')">$("Hrvatski (hr)")</a></li>
+  <li><a href="#" lang="it" data-lang-id="it" title="$_('Italian')">$("Italiano (it)")</a></li>
+  <li><a href="#" lang="pt" data-lang-id="pt" title="$_('Portuguese')">$("Português (pt)")</a></li>
+  <li><a href="#" lang="hi" data-lang-id="hi" title="$_('Hindi')">$("हिंदी (hi)")</a></li>
+  <li><a href="#" lang="sc" data-lang-id="sc" title="$_('Sardinian')">$("Sardu (sc)")</a></li>
+  <li><a href="#" lang="te" data-lang-id="te" title="$_('Telugu')">$("తెలుగు (te)")</a></li>
+  <li><a href="#" lang="uk" data-lang-id="uk" title="$_('Ukrainian')">$("Українська (uk)")</a></li>
+  <li><a href="#" lang="zh" data-lang-id="zh" title="$_('Chinese')">$("中文 (zh)")</a></li>
```

Comment:
isn't the point of these particular strings that they are explicitly _not_ localised? So that someone can locate their language regardless of the currently selected language.

Reply:
@hornc Yes! The `$(" ")` is a new syntax @cdrini came up with to explicitly _exclude_ phrases from translation. This way the new `detect-missing-i18n` script won't flag them as an error, which it would otherwise, but they will get to remain untranslated. 

I could see an argument for removing the existing `i18n` syntax for the `title` attribute, though, since I'm not sure why we would want to have those translated -- but can ask around to confirm.

Reply:
@rebecca-shoptaw thanks! I didn't even notice the difference between the `$()` and the `$_()` notation... I thought they were the same. Oops. I know now. It is subtle, but I guess the point is they don't add too much repetitive text. 

The WorldCat example maybe threw me, it's `$_("WorldCat")` to be translated, but `$('Library.link')` is not? I think `Library.Link` is correct to not be translated, but I'd put WorldCat in the same class (i.e. don't translate a brand)?

Anyway, sorry, my  initial comment was totally my confusion, thanks for pointing out the notation!

Reply:
Yeah; maybe let's use the more explicit `$# detect-missing-i18n-skip-line` at the end of the line. I think we might want to use that one all the time, since the difference between `$_` and `$(` is rather subtle!

Reply:
@hornc Update! Switched to `$# detect-missing-i18n-skip-line` so that the WARN syntax will just be for `i18n` typos 🙂 And after talking with Drini determined brands _should_ be translated, so switched Library.link to match WorldCat instead of vice versa.